src/coord/src/coord.rs

```diff
@@ -373,7 +332,7 @@ where
                 );
 
                 coord.update_catalog_view(
-                    MZ_VIEW_FOREIGN_KEYS.index_id,
+                    MZ_VIEW_FOREIGN_KEYS.id,
```

Comment:
I don't totally grok why this changed from index_id -> id

Reply:
Yeah the fact that it was `index_id` previously was actually a bit of a buglet in the code that sent `CreateLocalInput` commands for the built-in views before: https://github.com/MaterializeInc/materialize/blob/6bfa0584338263a9bfd1297d104f8276945f2ec3/src/coord/src/coord.rs#L335-L339

L338 there should have used `index.on_id`, not `id`. The only practical implication of this was needing to insert into e.g. `MZ_VIEW_FOREIGN_KEYS.index_id` rather than `MZ_VIEW_FOREIGN_KEYS.id`.